### PR TITLE
dev to main sync

### PR DESCRIPTION
**Pull request by shubhamsigdar1**
Not merge this now

Date: 23-11-2023

Developer Name: @shubhamsigdar1 , @sumitd94 

## Issue Ticket Number:- 

- https://github.com/Real-Dev-Squad/website-my/issues/66

## PR Number(s):-

- https://github.com/Real-Dev-Squad/website-my/pull/531

Is Under Feature Flag 
- [ ] Yes
- [X] No

Database changes
- [X] Yes
- [ ] No

Breaking changes (If your feature is breaking/missing something please mention pending tickets)
- [ ] Yes
- [x] No

Is Development Tested?

- [x] Yes
- [ ] No

Tested in staging?

- [x] Yes
- [ ] No

Add relevant Screenshot below

Screenshot 2023-11-24 at 3 49 02 PM

### CODE REVIEW

**Comment by shubhamsigdar1** [comment]
Date: 23-11-2023

Developer Name: @dhruv036 

## Issue Ticket Number:- 

- https://github.com/Real-Dev-Squad/website-my/issues/501

## PR Number(s):-

- https://github.com/Real-Dev-Squad/website-my/pull/506


Is Under Feature Flag 
- [X] Yes
- [ ] No

Database changes
- [ ] Yes
- [X] No

Breaking changes (If your feature is breaking/missing something please mention pending tickets)
- [ ] Yes
- [x] No

Is Development Tested?

- [x] Yes
- [ ] No

Tested in staging?

- [x] Yes
- [ ] No

Add relevant Screenshot below

**Comment by dhruv036** [comment]
> Date: 23-11-2023
> 
> Developer Name: @dhruv036
> 
> ## Issue Ticket Number:-
> * [Open RDS app from My-site #501](https://github.com/Real-Dev-Squad/website-my/issues/501)
> 
> ## PR Number(s):-
> * [Added Open RDS app from my-site #506](https://github.com/Real-Dev-Squad/website-my/pull/506)
> 
> Is Under Feature Flag
> 
> * [x]  Yes
> * [ ]  No
> 
> Database changes
> 
> * [ ]  Yes
> * [x]  No
> 
> Breaking changes (If your feature is breaking/missing something please mention pending tickets)
> 
> * [ ]  Yes
> * [x]  No
> 
> Is Development Tested?
> 
> * [x]  Yes
> * [ ]  No
> 
> Tested in staging?
> 
> * [x]  Yes
> * [ ]  No
> 
> Add relevant Screenshot below


https://github.com/Real-Dev-Squad/website-my/assets/56362412/f255b0f0-ec62-4115-8dba-75a591e9809d

File: app/templates/application.hbs [hunk 1]
```diff
@@ -1,16 +1,20 @@
 {{page-title "My | Real Dev Squad"}}
 
+<MobileDialog
+@dev ={{this.isDevMode}}
+@deviceType = {{this.toVisible}}
```
Comment (yesyash) [inline app/templates/application.hbs:5]: - we are passing a boolean here should we change the prop name from `deviceType` to something more representative of the value being passed?

File: app/styles/components/mobile-dialog.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.mobile-dialog {
+    display: block;
+    border-radius: 3px;
```
Comment (yesyash) [inline app/styles/components/mobile-dialog.css:3]: - as the rds design system has a base unit of `4px` please use multiples of 4 here (4, 8, 16, ...)

File: app/styles/components/mobile-dialog.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    text-align: center;
+    background-color: var(--form--bg);
+    border: none;
+    box-shadow: 4px 8px 8px var(--mobile-dialog-box-shadow);
+    & button {
+      padding: 0.5rem 1rem;
+      margin: 1rem;
+      font-size:medium;
+      border: none;
+      border-radius: 3px;
```
Comment (yesyash) [inline app/styles/components/mobile-dialog.css:17]: - please use multiples of 4 here.

**Review by yesyash — CHANGES_REQUESTED** [review]

File: app/components/mobile-dialog.hbs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{{#if @dev}}
+  {{#if @deviceType}}
+    {{#if this.toShow}}
+    <dialog data-test-mobile-dialog  class="mobile-dialog">
+        Open RDS in app
+      <br/>
+      <button id="mobile-dialog__close-button" type="button" {{on 'click' (fn this.closeDialog)}} >Cancle</button>
```
Comment (yesyash) [inline app/components/mobile-dialog.hbs:7]:
```suggestion
      <button id="mobile-dialog__close-button" type="button" {{on 'click' (fn this.closeDialog)}} >Cancel</button>
```

**Review by yesyash — CHANGES_REQUESTED** [review]

File: app/components/mobile-dialog.hbs [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{{#if @dev}}
+  {{#if @deviceType}}
+    {{#if this.toShow}}
+    <dialog data-test-mobile-dialog  class="mobile-dialog">
+        Open RDS in app
```
Comment (yesyash) [inline app/components/mobile-dialog.hbs:5]:
```suggestion
        Open in RDS App
```

**Review by yesyash — CHANGES_REQUESTED** [review]

**Review by sumitd94 — APPROVED** [review]

**Pull request merged**